docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -228,76 +232,42 @@ You can update to the Grafana Mimir Helm chart from the Cortex Helm chart.
    b. Use `mimirtool` to update the configuration.
 
    ```bash
-   mimirtool config convert --yaml-file cortex.yaml
+   mimirtool config convert --yaml-file cortex.yaml --yaml-out mimir.yaml
+   ```
+
+   c. Clean up the generated YAML configuration.
+   You have to remove some fields that are generated by `mimirtool config convert` for the Helm chart installation, because the
+   mimir-distributed Helm chart has already set the default value for them. Use the following script to clean up
+   those fields:
+
+   ```bash
+   # require yq v4
```

Comment:
IIRC there are two different YQs. Can you include a link to the one we're using? may also help to add installing it as a separate step?

Reply:
The latest version is v4. I will add a step in the "Before you begin" section to tell the user to install yq v4. However, I feel that adding installing guide for yq is out of the scope of this guide.